src/pages/workspaces/workspace/Data.js

```diff
@@ -223,18 +223,27 @@ const BucketContent = _.flow(
     activeStyle: { backgroundColor: colors.accent(0.2), cursor: 'copy' },
     onDropAccepted: uploadFiles
   }, [({ openUploader }) => h(Fragment, [
-    div([
-      _.map(({ label, target }) => {
-        return h(Fragment, { key: target }, [
-          makeBucketLink({ label, target, onClick: () => load(target) }),
-          ' / '
+    div({ style: { display: 'flex', justifyContent: 'space-between' } }, [
+      div([
+        _.map(({ label, target }) => {
+          return h(Fragment, { key: target }, [
+            makeBucketLink({ label, target, onClick: () => load(target) }),
+            ' / '
+          ])
+        }, [
+          { label: 'Files', target: '' },
+          ..._.map(n => {
+            return { label: prefixParts[n], target: _.map(s => `${s}/`, _.take(n + 1, prefixParts)).join('') }
+          }, _.range(0, prefixParts.length))
         ])
-      }, [
-        { label: 'Files', target: '' },
-        ..._.map(n => {
-          return { label: prefixParts[n], target: _.map(s => `${s}/`, _.take(n + 1, prefixParts)).join('') }
-        }, _.range(0, prefixParts.length))
+      ]),
+
+      div({ style: { justifyContent: 'flex-end' } }, [
+        h(Link, { href: `https://seqr.broadinstitute.org/workspace/${namespace}/${workspace.workspace.name}` },
+          ['Analyze in Seqr ', icon('pop-out', { size: 14 })]
+        )
       ])
```

Comment:
could it cause any problems that all Terra envs point to the same production Seqr instance? For instance, if we have a workspace with the same namespace+name in both alpha and prod, would that cause any problems in Seqr?

Reply:
Hmm that's a good question. I don't know much about how Seqr works so I'll ask Janki about that.